tests/Remove-DbaAgentJob.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+            $params.Count - $defaultParamCount | Should Be $paramCount
+        }
+    }
+}
+Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
+    Context "Command removes jobs" {
+        BeforeAll {
+            $null = New-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_daily -FrequencyType Daily -FrequencyInterval Everyday -Force
+            $null = New-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_testjob -Schedule dbatoolsci_daily
+            $null = New-DbaAgentJobStep -SqlInstance $script:instance2 -Job dbatoolsci_testjob -StepId 1 -StepName dbatoolsci_step1 -Subsystem TransactSql -Command 'select 1'
+            $null = Start-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_testjob
+        }
+        AfterAll {
+            if (Get-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_daily) { Remove-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_daily }
+        }
+        $null = Remove-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_testjob
+        It "Should have deleted job: dbatoolsci_testjob" {
+            (Get-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_testjob) | Should BeNullOrEmpty
+        }
+        It "Should have deleted schedule: dbatoolsci_daily" {
+            (Get-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_daily) | Should BeNullOrEmpty
+        }
+        It "Should have deleted history: dbatoolsci_daily" {
+            (Get-DbaAgentJobHistory -SqlInstance $script:instance2 -Job dbatoolsci_testjob) | Should BeNullOrEmpty
+        }
+    }
+    Context "Command removes job but not schedule" {
+        BeforeAll {
+            $null = New-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_weekly -FrequencyType Weekly -FrequencyInterval Everyday -Force
+            $null = New-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_testjob_schedule -Schedule dbatoolsci_weekly
+            $null = New-DbaAgentJobStep -SqlInstance $script:instance2 -Job dbatoolsci_testjob_schedule -StepId 1 -StepName dbatoolsci_step1 -Subsystem TransactSql -Command 'select 1'
+        }
+        AfterAll {
+            if (Get-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_weekly) { Remove-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_weekly }
+        }
+        $null = Remove-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_testjob_schedule -KeepUnusedSchedule
+        It "Should have deleted job: dbatoolsci_testjob_schedule" {
+            (Get-DbaAgentJob -SqlInstance $script:instance2 -Job dbatoolsci_testjob_schedule) | Should BeNullOrEmpty
+        }
+        It "Should not have deleted schedule: dbatoolsci_daily" {
+            (Get-DbaAgentSchedule -SqlInstance $script:instance2 -Schedule dbatoolsci_weekly) | Should Not BeNullOrEmpty
+        }
+        Context "Command removes job but not history" {
```

Comment:
This needs to be a new context script block. As-is this is within the context above "Command removes job but not schedule". Just missing a closing curly brace.

Reply:
Not sure if that is why the test is failing but worth updating to just make sure.